src/main/java/florizz/command/InfoCommand.java

```diff
@@ -5,18 +5,25 @@
 import florizz.objects.Bouquet;
 
 import java.util.ArrayList;
+import java.util.logging.ConsoleHandler;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 public class InfoCommand extends Command{
-    String flowerName;
+    private static final Logger logger = Logger.getLogger("InfoLog");
+    private ConsoleHandler ch = new ConsoleHandler();
+    private String flowerName;
```

Comment:
line 13: private static final Logger logger = Logger.getLogger(InfoCommand.class.getName());

just to standardize the logger name